src/components/POLine/POLineInvoices/POLineInvoicesContainer.js

```diff
@@ -7,23 +7,34 @@ import { stripesConnect } from '@folio/stripes/core';
 import { Accordion } from '@folio/stripes/components';
 
 import {
-  INVOICE_LINES,
-  INVOICES,
+  INVOICE_LINES, INVOICES,
   RECEIVING_HISTORY,
 } from '../../Utils/resources';
 
 import POLineInvoices from './POLineInvoices';
 
-const POLineInvoicesContainer = ({
-  accordionId,
-  label,
-  resources,
-  vendors,
-}) => {
+const POLineInvoicesContainer = ({ accordionId, label, resources, vendors, mutator }) => {
   const lineInvoices = get(resources, ['invoices', 'records'], []);
   const invoiceLines = get(resources, ['invoiceLines', 'records'], []);
   const pieces = get(resources, ['pieces', 'records'], []);
 
+  useEffect(() => {
+    mutator.invoiceLines.reset();
```

Comment:
don't you need to check `lineId` change in effect?

Reply:
I thought it goes as a second argument, `}, [lineId]);` line 36

Reply:
I did this